package.json

```diff
@@ -3,6 +3,7 @@
   "version": "1.2.0",
   "scripts": {
     "publish:json-schemas": "node json-schemas/publish.js",
+    "lint:data": "prettier --check **/*.{json,yml}",
```

Comment:
I think whenever we add an NPM script to check formatting we should add one to apply the formatting (`prettier --write [args]`) so that developers can quickly fix up their branch if this script fails.

Also, I think it's common in JS projects to have separate scripts for linting vs formatting so I'd maybe change this script to be called something like `format:check` (or `format:check:data`?) although I'm not too opinionated on this so feel free to leave it as-is. See the [ably-js setup](https://github.com/ably/ably-js/blob/main/package.json#L101) for an example of what I mean.

Reply:
Fair point. I'll admit I tend to be a little bit too IDE-bound in terms of my preference to fix formatting issues 'by hand' at times. 😳 

Hopefully this is what you were wanting: [2034b06](https://github.com/ably/ably-common/pull/138/commits/2034b06d0e94a712756666929d3bc4bc1d11b3e2)

Reply:
Yeah, that's exactly what I wanted, thanks 🙂